io.github.bruttazz.FirefoxDevFlatpak.yml

```diff
@@ -0,0 +1,92 @@
+id: io.github.bruttazz.FirefoxDevFlatpak
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: firefox
+
+build-options: 
+  build-args:
+    - --share=network
```

Comment:
this is not allowed and will not work.

Reply:
But it's just a build arg right??

Reply:
what don't you understand by "this is not allowed and will not work"?